api/v1beta1/agentserviceconfig_types.go

```diff
@@ -34,6 +34,8 @@ type OSImage struct {
 	Url string `json:"url"`
 	// rootFSUrl specifies the path to the root filesystem.
 	RootFSUrl string `json:"rootFSUrl"`
+	// The CPU architecture of the image (x86_64/arm64/etc).
+	CPUArchitecture string `json:"cpu_architecture"`
```

Comment:
I'm not sure I follow this comment, but I think it's because `CPU` is an acronym. Starting with upper case is just to make it public. Tell me if I missed something